src/keyword-value.js

```diff
@@ -22,7 +22,7 @@
     this.cssString = value;
   }
 
-  KeywordValue.StyleValueKeyword = ['initial', 'inherit', 'revert', 'unset'];
+  KeywordValue.StyleValueKeyword = ['auto', 'initial', 'inherit', 'revert', 'unset'];
```

Comment:
As we are going for the keyword soup approach, I don't think "isGlobalKeyword" is as appropriate as before, nor do I think that the following comment is still applicable.  I stand to be corrected though.
"I think the keywords for particular types should go in the classes used for those purposes."

So, I'll just let this be for now, unless otherwise advised.